src/index.ts

```diff
@@ -88,4 +88,13 @@ export class NoSqlEasy implements IRepository {
   async getSizeCollection<T>(collection: string, options?: Options<T>): Promise<number> {
     return this.repository.getSizeCollection<T>(collection, options);
   }
+
+  getPaginatedCollection<T, F>(
+    collection: string,
+    queryParams?: any,
+    FilterClass?: (new () => F) | undefined,
+    orderBy?: OrderBy<T>,
+  ): Promise<T[]> {
+    return this.repository.getPaginatedCollection<T, F>(collection, queryParams, FilterClass, orderBy);
+  }
```

Comment:
It's necessary to use async in this method

Reply:
Based on the documentation below and discussions with the team, we decided to remove all _async_ keywords from index.ts:
- https://dev.to/somedood/best-practices-for-es2017-asynchronous-functions-async-await-39ji;
- https://eslint.org/docs/rules/require-await#disallow-async-functions-which-have-no-await-expression-require-await;